setup-environment-internal

```diff
@@ -247,10 +247,10 @@ ln -sf "${MANIFESTS}" "${OEROOT}"/layers/
 
 DISTRO_DIRNAME=$(echo "${DISTRO}" | sed 's#[.-]#_#g')
 
-LMP_VER=$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)
-if [ -v LMP_VER_DEV ]; then
+LMP_VERSION_NUM="${LMP_VERSION_NUM:-$(git --git-dir ${MANIFESTS}/.git describe HEAD --tags --abbrev=0)}"
```

Comment:
We should be fine to update it to LMP_VERSION_CACHE now, right?

Reply:
I will do it